src/content/core/components/namespaces.mdx

```diff
@@ -31,18 +46,27 @@ To share a namespace go to the Okteto dashboard, select the namespace you want t
 
 <p align="center"><Image src={require('@site/static/img/share.png').default} alt="share a namespace" width="850" /></p>
 
-In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto dashboardll update the necessary permissions and notify your teammates via email.
+In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto dashboard will update the necessary permissions and notify your teammates via email.
 
-You can sleep your namespace from the command line with `okteto namespace sleep [name]`.
 
-When you want to wake your namespace, you can use the command `okteto namespace wake [name]`.
+## Transfer your namespace
+
+Okteto administrators have the ability to transfer non-default namespaces between users. This process can be achieved through the Okteto dashboard.
+
+To transfer a namespace to a new owner, go to the Okteto dashboard -> Admin -> Namespaces then locate the namespace that you would like to transfer. Click on the three dots `...` to display the namespace menu. From the namespace menu select Change Owner.
 
-You can sleep your namespace from Okteto dashboardself by clicking on Sleep dialog.
+<p align="center"><Image src={require('@site/static/img/namespace-change-owner.png').default} alt="share a namespace" width="650" /></p>
+
+When the Change Owner modal appears, type the email address of the team member who you'd like the transfer the namespace to. Once you press confirm, ownership of that namespace will be immediately transferred.
+If you wish, you can keep the current owner as a member of the namespace by selecting the option in the Change Owner modal. If this is not selected the current owner will immediately lose access.
+
+<p align="center"><Image src={require('@site/static/img/namespace-transfer-modal.png').default} alt="share a namespace" width="700" /></p>
 
-In order to wake the namespace, you can click on Wake dialog.
 
 :::tip
 
-SaaS single tenant and Self-Hosted customers can specify a global, default threshold for automatically sleeping a namespace in the [garbage collection](reference/helm-chart-values.mdx/#gc) configuration.
+User specific secrets will not transfer along with a namespace. If the namespace being transferred has user specific secrets, please note that future build or destroy actions may fail.
 
-:::
+New user specific secrets can be added to a transferred namespace if desired to avoid future failures.
```

Comment:
```suggestion
New user specific variables can be added to a transferred namespace if desired to avoid future failures.
```

Reply:
Same comment about "variables added to a transferred namespace". Namespaces don't have variables.

To avoid future issues, the best way would be to make use of cluster variables or specify the value at deployment time through the UI or with `--var` parameter in the CLI. 

I think we need to think about the future of user specific variables. They are problematic for sharing namespace, transfer ownership or admin performing actions in any namespace scenarios

Reply:
I applied this change though I'm not sure if I should have 😅 

@ifbyol is your comment in opposition to this change given the dynamic you're referring to?

Reply:
Rename from secrets to variable is fine. My comment is more about the fact of adding variables to a namespace. That cannot be done. The namespace cannot have variables

Reply:
Thanks for the clarification. How would you rephrase the final section of the Tip? Should that be removed all together?

Reply:
instead of 

```
User specific variables will not transfer along with a namespace. If the namespace being transferred has user specific variables, please note that future build or destroy actions may fail.

New user specific variables can be added to a transferred namespace if desired to avoid future failures. Details on [adding variables can be found here](manifest/okteto-variables.mdx).
```

We could write something like:

```
User specific variables will not transfer along with a namespace. If some deployed development environment within the transferred namespace was deployed using user specific variables, please note that future build or destroy actions may fail.

New user specific variables can be added to the new owner. Details on [adding variables can be found here](manifest/okteto-variables.mdx). Also, consider to move user specific variables to cluster variables (link to cluster variables)
```

Also, I think it might be relevant to highlight the fact that has to be the new owner the one that has to add the new variables to themselves. The admin transferring the namespace cannot add user variables to other user different than themselves